src/main/java/dev/snowdrop/release/services/GitService.java

```diff
@@ -110,16 +110,14 @@ public void commitAndPush(String commitMessage, GitConfig config, FileModifier..
                         final var rmCommand = g.rm();
                         files.forEach(file -> {
                             final var path = file.getAbsolutePath().replace(repository.getAbsolutePath() + "/", "");
-                            if (untracked.contains(path) || statusChanged.contains(path)) {
-                                // only add file to be committed if it's part of the modified set or untracked
-                                LOG.infof("Added %s", path);
-                                addCommand.addFilepattern(path);
-                                hasAdd[0] = true;
-                            }
                             if (removed.contains(path) || missing.contains(path)) {
```

Comment:
can you add a little comment here please?

Reply:
done.